build-scripts/build_all_remediation_roles.py

```diff
@@ -59,6 +59,9 @@ def main():
     extension = args.extension
     template = args.template
 
+    if path_base.startswith("ssg-"):
+        path_base = path_base[len("ssg-"):]
```

Comment:
I would say either save the `ssg-` prefix to a variable, or hardcode trim `path_base[3:]`.

Reply:
I have saved the prefix in a new variable.